src/components/Votes/ApprovalCastVoteButton/ApprovalCastVoteButton.tsx

```diff
@@ -77,6 +87,58 @@ export default function ApprovalCastVoteButton({
   return (
     <VStack className="flex-shrink-0">
       <VStack alignItems="items-stretch">
+        {isReady && (
+          <div className="pt-3">
+            <TooltipProvider>
+              <Tooltip>
+                <TooltipTrigger className="w-full flex items-center justify-center gap-1 text-primary font-medium cursor-help">
+                  <span className="flex items-center">
+                    Proposal voting power{"\u00A0"}
+                    <TokenAmountDisplay amount={votingPower.totalVP} />
+                    <Image
+                      src={icons.info}
+                      alt="Info"
+                      width={16}
+                      height={16}
+                      className="ml-1"
+                    />
+                  </span>
+                </TooltipTrigger>
+                <TooltipContent
+                  side="top"
+                  align="center"
+                  className="bg-neutral p-4 rounded-lg border border-line shadow-newDefault w-[calc(100vw-32px)] sm:w-[400px]"
+                >
+                  <div className="flex flex-col gap-4">
+                    <div>
+                      <div className="text-lg font-medium">
+                        Proposal launched
+                      </div>
+                      <div className="text-lg font-medium">
+                        {new Intl.DateTimeFormat("en-US", {
+                          month: "short",
+                          day: "numeric",
+                          year: "numeric",
+                          hour: "numeric",
+                          minute: "numeric",
+                          timeZoneName: "short",
+                        }).format(new Date(proposal.startTime ?? ""))}
```

Comment:
Not necessary for getting this PR merged, but we should make some helper fns for formatting dates

Reply:
We have lots of one-off date formatters in the codebase so could be cool for someone in another ticket to go through and standardize them